backend/muster-roll/src/main/resources/db/migrate.sh

```
@@ -1,3 +1,11 @@
-flyway -url=$DB_URL -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS -baselineOnMigrate=true   -outOfOrder=true migrate
\ No newline at end of file
+#!/bin/bash
+baseurl=$DB_URL
+echo "the baseurl : $DB_URL"
+schemasetter="?currentSchema="
+schemas=$SCHEMA_NAME
+echo "the schemas : $schemas"
+for schemaname in ${schemas//,/ }
+do
+    echo "the schema name : ${baseurl}${schemasetter}${schemaname}"
+    flyway -url=${baseurl}${schemasetter}${schemaname}  -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS -baselineOnMigrate=true -outOfOrder=true migrate
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)